Data/Sys/GameSettings/R92.ini

```diff
@@ -0,0 +1,14 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 4
+EmulationIssues = Distorted sound effects on DSP HLE. Use DSP LLE to fix.
```

Comment:
According to the wiki, HLE doesn't work at all after the new Zelda HLE merge. I'm guessing it works fine in the newest build, though. Either way, this description is out of date and should not be in the INI. If there indeed is a problem with HLE not working at all, we should force enable LLE instead.